.prettierignore

```diff
@@ -5,10 +5,20 @@
 # We decided in the team to not format scss files
 *.scss
 
+src/static
+*.svg
+*.jpg
+*.pdf
+*.png
+*.rtf
+.gitignore
+
 # Ignore auto generated files
 node_modules
 package-lock.json
 lib
 dist
 src/components/00-materials/icons
+src/components/00-materials/icons-raw
```

Comment:
Why are you excluding the raw icons?

Reply:
See the linked issue for your questions

Reply:
if you anyway exclude all avgs, there is no need to add those 2 pathes here.

Reply:
true